services/app-api/serverless.yml

```diff
@@ -159,6 +159,19 @@ functions:
               paths:
                 state: true
                 reportId: true
+  deleteReportData:
+    handler: handlers/banners/delete.deleteReportData
+    events:
+      - http:
+          path: reportData/{state}/{reportId}
```

Comment:
Are both these functions being kicked off by the same path? If they should always run together, why not have them as a single function?